src/test/java/com/epam/aidial/cfg/domain/validator/EndpointValidatorTest.java

```diff
@@ -0,0 +1,76 @@
+package com.epam.aidial.cfg.domain.validator;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+class EndpointValidatorTest {
+
+    @Test
+    void isValidUrlPath_shouldReturnTrueForSimplePaths() {
+        assertTrue(EndpointValidator.isValidUrlPath("/api"));
+        assertTrue(EndpointValidator.isValidUrlPath("/"));
+        assertTrue(EndpointValidator.isValidUrlPath("/v1"));
+    }
+
+    @Test
+    void isValidUrlPath_shouldReturnTrueForNestedPaths() {
+        assertTrue(EndpointValidator.isValidUrlPath("/api/123"));
+        assertTrue(EndpointValidator.isValidUrlPath("/api/"));
+    }
+
+    @Test
+    void isValidUrlPath_shouldReturnTrueForPathsWithoutLeadingSlash() {
+        assertTrue(EndpointValidator.isValidUrlPath("api"));
+        assertTrue(EndpointValidator.isValidUrlPath("somePath/"));
+    }
+
+    @Test
+    void isValidUrlPath_shouldReturnTrueForPathsWithTrailingSlash() {
+        assertTrue(EndpointValidator.isValidUrlPath("/somePath/"));
+    }
+
+    @Test
+    void isValidUrlPath_shouldReturnTrueForPathsWithQueryParameters() {
+        assertTrue(EndpointValidator.isValidUrlPath("/somePath?param1=val1;param2=val2"));
```

Comment:
not for our case. we should not allow query param in `endpoints` props in config

Reply:
fixed